docs/ecosystem/testing-infrastructure/memtest.md

```diff
@@ -0,0 +1,142 @@
+# Dasharo Compatibility: Memtest
+
+## Test cases
+
+### MEM001.001 Memtest availability
+
+**Test description**
+
+This test aims to verify that the Memtest entry is available in DUT.
+
+**Test configuration data**
+
+1. `BOOT_MENU_KEY` = `ESC`
+2. `BOOT_MENU_STRING` = `Press ESC for boot menu`
+
+**Test setup**
+
+1. Proceed with the [Generic test setup: firmware](../generic-test-setup.md/#firmware).
+
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Check if `Payload [memtest]` is available in the boot menu.
+
+**Expected result**
+
+The `Payload [memtest]` option should be visible under `Select boot device:`.
+
+### MEM001.002 Enter Memtest
+
+**Test description**
+
+This test aims to verify that the DUT enters the Memtest boot option.
+
+**Test configuration data**
+
+1. `BOOT_MENU_KEY` = `ESC`
+2. `BOOT_MENU_STRING` = `Press ESC for boot menu`
+
+**Test setup**
+
+1. Proceed with the [Generic test setup: firmware](../generic-test-setup.md/#firmware).
+
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the key with a proper number for `Payload [memtest]`.
+5. Check if `Memtest86+` is available.
+
+**Expected result**
+
+The `Memtest86+` is visible at the top of the output.
```

Comment:
I think that we should change that description and write about entering the `Memtest86+` option.